generator/src/generator/namespace.rs

```diff
@@ -904,18 +905,45 @@ impl<'ns, 'c> NamespaceGenerator<'ns, 'c> {
                         },
                         _ => (),
                     }
-                    match field {
-                        _ => {
-                            let rust_field_name = to_rust_variable_name(field.name().unwrap_or("<unnamed field>"));
-                            outln!(out, "// TODO: deserialize {}", rust_field_name);
-                        }
-                    }
+
+                    let from = if !seen_complete_header {
+                        assert_eq!(field.size(), Some(1));
+                        "&[header.minor_opcode]"
```

Comment:
Ah, yes. This happens when I "complain" after having read only half of the first commit. Sorry! :-)

Reply:
No worries.